src/main/groovy/edu/oregonstate/mist/slashcourses/core/Instructor.groovy

```diff
@@ -0,0 +1,10 @@
+package edu.oregonstate.mist.slashcourses.core
+
+/**
+ * Instructor Class
+ */
+class Instructor {
+    Integer instructor_id
```

Comment:
The convention in java is to use: camelCase for class properties. In this class it should be: firstName and lastName.